integration/tests/form-behavior.spec.ts

```diff
@@ -39,37 +39,6 @@ test.describe('form behavior validation', () => {
 		});
 	});
 
-	// The input value string is automatically reduced if input has maxlength
```

Comment:
For a normal input, setting the `maxlength` attribute will automatically limit the number of characters to its value so this test is redundant.